libs/libsodium/Makefile

```diff
@@ -52,6 +52,7 @@ endef
 
 CONFIGURE_ARGS+= \
 	--disable-ssp \
+	--without-pthreads \
```

Comment:
As I understand CC support limited to bug-fixes and security fixes. So I made new [pull request without CC support](https://github.com/openwrt/packages/pull/3115).